google-cloud-bigtable-stats/src/main/java/com/google/cloud/bigtable/stats/BuiltinViewConstants.java

```diff
@@ -183,4 +173,29 @@ class BuiltinViewConstants {
           AGGREGATION_WITH_MILLIS_HISTOGRAM,
           ImmutableList.of(
               PROJECT_ID, INSTANCE_ID, APP_PROFILE, METHOD, CLIENT_NAME, CLUSTER, ZONE, TABLE));
+
+  static final View PER_CONNECTION_ERROR_COUNT_VIEW =
+      View.create(
+          View.Name.create("bigtable.googleapis.com/internal/client/per_connection_error_count"),
+          "Distribution of counts of channels per 'error count per minute'.",
+          PER_CONNECTION_ERROR_COUNT,
+          PER_CONNECTION_ERROR_COUNT_AGGREGATION,
+          ImmutableList.of(
```

Comment:
Same here, I don't think we need to tag the view with GKE / GCE tag keys. These values are unknown when we populate the metrics.